menu/src/components/WarnPage/WarnPage.tsx

```diff
@@ -1,40 +1,51 @@
-import React, { useRef, useState } from "react";
-import { Box, Fade, Typography } from "@mui/material";
-import makeStyles from '@mui/styles/makeStyles';
-import { useNuiEvent } from "../../hooks/useNuiEvent";
-import { useTranslate } from "react-polyglot";
+import React, {useRef, useState} from "react";
```

Comment:
this page is completely fucked, if it's too hard to migrate just make a new one, keeping in mind the dismiss pulse animation, more or less the size, and obviously the bg color and transparency should be exactly the same